klister.cabal

```diff
@@ -0,0 +1,120 @@
+cabal-version: 2.2
+
+name:           klister
```

Comment:
nit: In general, I prefer non-aligned code. Consider what the diff would look like if we added the famous `really-long-cabal-field-everything-needs-to-be-reindented-now: true` field to this config.